spec.md

```diff
@@ -436,6 +436,8 @@ These limits apply for messages generated by both COs and plugins.
 | 128 bytes  | string              |
 | 4 KiB      | map<string, string> |
 
+Please note that for any length of paths or id, 128 bytes will still be the preferrable size for backward compatibility.
```

Comment:
Remove this and just include it in each override as specified below.

Reply:
Done